apps/test/unit/code-studio/components/progress/ScriptOverviewTopRowTest.jsx

```diff
@@ -27,6 +28,31 @@ const defaultProps = {
   showAssignButton: true
 };
 
+const sampleLessons = [
```

Comment:
It would be great to use the sample lessons from the sample data you created in step 2 here.